pkg/libcose/Kconfig

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig PACKAGE_LIBCOSE
+    bool "LibCose CBOR Object Signing and Encryption package"
+    depends on TEST_KCONFIG
+    depends on MODULE_RANDOM
+    select PACKAGE_NANOCBOR
+    select MODULE_LIBCOSE_CRYPT
+    help
+        CBOR Object Signing and Encryption package based on RFC8613.
+
+if PACKAGE_LIBCOSE
+
+config MODULE_LIBCOSE_CRYPT_C25519
+    bool "COSE use C25519 as backend"
+    depends on TEST_KCONFIG
+    depends on PACKAGE_C25519
+
+config MODULE_LIBCOSE_CRYPT_HACL
+    bool "COSE use HACL as backend"
+    depends on TEST_KCONFIG
+    depends on PACKAGE_HACL
+
+config MODULE_LIBCOSE_CRYPT_MONOCYPHER
+    bool "COSE use MONOCYPHER as backend"
+    depends on TEST_KCONFIG
+    depends on PACKAGE_MONOCYPHER
```

Comment:
Choosing a backend looks more like a `choice` to me. Could it be that these are mutually exclusive?

Reply:
Apparently not, as in the documentation says you can have many at once (I didn't look into it but I think they just expose different functions).

Maybe backend is not the right wording if that is the case.

Reply:
Any resolution?

Reply:
Sounds right, it seems one can select more than one at a time.